src/lib.rs

```diff
@@ -176,7 +176,7 @@ macro_rules! impl_elastic_array {
 			}
 
 			pub fn clear(&mut self) {
-				self.raw = $dummy::Arr(unsafe { $crate::core_::mem::uninitialized() });
+				self.raw = $dummy::Arr([T::default(); $size]);
```

Comment:
Out of scope of this PR: I would prefer renaming `size` -> `len` because `size == size_of<T> * len`

Reply:
I agree. There's plenty of things to fix here, but I hope we can switch to `smallvec` in `parity-common` asap so probably not worth it atm.